cpp/test/Ice/custom/CustomBuffer.h

```diff
@@ -53,6 +59,19 @@ namespace Test
             return *this;
         }
 
+        CustomBuffer& operator=(CustomBuffer&& o)
+        {
+            if (_buf)
```

Comment:
We can remove the if check like in `MyByteSeq::operator=(`

Reply:
Fixed